languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/hunspell/ar_AE.aff

```diff
@@ -0,0 +1 @@
+ar.aff
```

Comment:
As these are all links, I don't think these files (links) should be needed. The files should only be needed when the spelling of the country variants is actually different, or am I missing something? Does something not work when these links don't exist?